lib/jxl/splines.cc

```diff
@@ -61,6 +62,9 @@ void DrawGaussian(Image3F* const opsin, const Rect& opsin_rect,
                   const float sigma, std::vector<int32_t>& xs,
                   std::vector<int32_t>& ys,
                   std::vector<float>& local_intensity_storage) {
+  // Sanity check sigma, inverse sigma and intensity
+  if (!(std::isfinite(sigma) && std::isfinite(1.0f / sigma) &&
+      std::isfinite(intensity))) return;
```

Comment:
With the check taking up two lines, we would prefer to have braces around the `return`: https://google.github.io/styleguide/cppguide.html#Conditionals

```suggestion
  if (!(std::isfinite(sigma) && std::isfinite(1.0f / sigma) &&
      std::isfinite(intensity))) {
    return;
  }
```